src/core/thread/key_manager.hpp

```diff
@@ -61,6 +62,124 @@ namespace ot {
  * @{
  */
 
+/**
+ * This class represents Security Policy Rotation and Flags.
+ *
+ */
+OT_TOOL_PACKED_BEGIN
+class SecurityPolicy : public otSecurityPolicy
+{
+public:
+    enum : uint16_t
+    {
+        kMinKeyRotationTime     = 1,   ///< The minimum Key Rotation Time in hours.
+        kDefaultKeyRotationTime = 672, ///< Default Key Rotation Time (in unit of hours).
+    };
+
+    enum : uint8_t
+    {
+        kThread11FlagsLength =
+            OT_THREAD_1_1_SECURITY_POLICY_FLAGS_LENGTH, ///< The Thread 1.1 Security Policy Flags length.
+        kThread12FlagsLength =
+            OT_THREAD_1_2_SECURITY_POLICY_FLAGS_LENGTH, ///< The Thread 1.2 Security Policy Flags length.
+    };
```

Comment:
Can this be `private`?

Reply:
`kThread11FlagsLength` is used by `SecurityPolicyTlv` to validation itself.

Reply:
I moved the definition to `SecurityPolicyTlv` and declared it as private.